post/views.py

```diff
@@ -62,3 +64,35 @@ def update_post(request, id):
             return redirect("/")
     context = {"form": form, "post": post, "success_message": success_message}
     return render(request, "post/update_post.html", context)
+
+class AddCommentView(CreateView):
+      model = Comment
+      form_class = CommentForm
+      template_name = 'post/add_comment.html'
+      success_url = reverse_lazy('')
+      
+      def form_valid(self, form):
+          form.instance.post_id = self.kwargs['pk']
+          return super().form_valid(form)
+
```

Comment:
why do you use a class based view? why reverse lazy?

Reply:
I double checked with Anna , she said we don't have to use the same view, it's ok... reverse lazy means redirecting to _ page

Reply:
I reverse lazy back to home.. same as post , so it's consistent